templates/form/keywordInput.tpl

```diff
@@ -46,13 +46,13 @@
 		{rdelim});
 		</script>
 		<span id="{$FBV_id|escape}-localization-popover-container{$uniqId}" class="localization_popover_container pkpTagit">
-			<ul class="localization_popover_container localizable {if $formLocale != $currentLocale} flag flag_{$formLocale|escape}{/if}" id="{$formLocale|escape}-{$FBV_id|escape}{$uniqId}">
+			<ul class="localization_popover_container localizable {if $formLocale != $currentLocale} flag flag_{$formLocale|escape}{/if}" id="{$formLocale|escape|escape:jqselector}-{$FBV_id|escape}{$uniqId}">
```

Comment:
right. To be honest I can't remember why I escaped this line.. I just removed this extra escape and everything seems to be working well.